recipe/recipe.yaml

```diff
@@ -0,0 +1,577 @@
+context:
+  version: "7.8.3"
+
+recipe:
+  name: suitesparse
+  version: ${{ version }}
+
+source:
+  url: https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/v${{ version }}.tar.gz
+  sha256: ce39b28d4038a09c14f21e02c664401be73c0cb96a9198418d6a98a7db73a259
+
+build:
+  number: 2
+  string: ss${{ version | replace('.', '') }}_h${{ hash }}
+
+outputs:
+  - package:
+      name: libsuitesparseconfig
+    about:
+      license: BSD-3-Clause
+      license_file: LICENSE.txt
+    build:
+      script:
+        # rattler-build adds `.sh` or `.bat` based on platform
+        file: build-one
+        env:
+          SUBPKG_DIR: SuiteSparse_config
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - ${{ compiler('fortran') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
+        - pkg-config
+      host:
+        - if: osx
+          then:
+            - llvm-openmp
+        - if: linux
+          then:
+            - libgomp
+        - libblas
+        - libcblas
+        - liblapack
+      ignore_run_exports:
+        from_package:
+          - libblas
+          - libcblas
+          - liblapack
+          - ${{ fortran_compiler }}
+      run_exports:
+        - ${{ pin_subpackage('libsuitesparseconfig') }}
+    tests:
+      - package_contents:
+          lib:
+            - suitesparseconfig
+  - package:
+      name: libamd
+      version: 3.3.3
+    about:
+      license: BSD-3-Clause
+      license_file: AMD/Doc/License.txt
+    build:
+      script:
+        file: build-one
+        env:
+          SUBPKG_DIR: AMD
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - ${{ compiler('fortran') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
+        - pkg-config
+      host:
+        - ${{ pin_subpackage('libsuitesparseconfig', exact=True) }}
+      run_exports:
```

Comment:
I'm confused? Why is there a run_exports section in here? What does it do? 

Hint: I have not created complex recipes with v1 syntax.

Reply:
Each library output should have its own `run_exports` because the idea behind this split is that ~everything depending on `suitesparse` now should migrate to depending directly on the subset of libraries they actually use (`libumfpack`, etc.). If the question is why it's under `requirements`, v1 moves `build.run_exports` to `requirements.run_exports`.